### ISSUE TITLE
Dreame X10 Pro and HA - cleaning start doesn't work

### ISSUE THREAD

**Issue by Andrey-Volkov77** [Tasshack/dreame-vacuum#249]
Home Assistant 2023.10.3
Supervisor 2023.10.0
Operating System 11.0

Dreame X10 Pro:
dreame.vacuum.r2215
by Dreametech™
Firmware: 4.3.6_1411

Dreame Vacuum [v2.0.0b6]

The vacuum cleaner has been added to the integration, all sensors are working, the map is visible. But the control does not start
I can give you teamviewer/anydesk

![photo_2023-10-18_15-35-23](https://github.com/Tasshack/dreame-vacuum/assets/62943818/9b3a40f8-ab6e-4f2a-8d62-6a035c4369f6)
![photo_2023-10-18_15-35-35](https://github.com/Tasshack/dreame-vacuum/assets/62943818/68ba1fa0-b898-4663-83f5-c89acd2d3945)
![photo_2023-10-18_15-35-40](https://github.com/Tasshack/dreame-vacuum/assets/62943818/29373858-658d-4a4f-bd20-6da32c64537f)

https://github.com/Tasshack/dreame-vacuum/assets/62943818/81e9bcfd-005b-41c4-a39f-0946348a2de7

**Comment by Andrey-Volkov77** [comment]
https://github.com/Tasshack/dreame-vacuum/assets/62943818/303370d2-bb16-44eb-b3a6-7f78d2ec90e3

**Comment by Andrey-Volkov77** [comment]
no, no settings can be made. The vacuum cleaner is blocked, mihome says region blocked. They wrote on the forum that even a blocked vacuum cleaner will work through HA. this option doesn't work now

> 18 окт. 2023 г., в 16:15, Tasshack ***@***.***> написал(а):
> 
> Can you check that you can change any settings from integration? If not, that means vacuum does not respond to you because it is region locked.

**Comment by Shm1ks** [comment]
> Home Assistant 2023.10.3 Supervisor 2023.10.0 Operating System 11.0
> 
> Dreame X10 Pro: dreame.vacuum.r2215 by Dreametech™ Firmware: 4.3.6_1411
> 
> Dreame Vacuum [v2.0.0b6]
> 
> The vacuum cleaner has been added to the integration, all sensors are working, the map is visible. But the control does not start I can give you teamviewer/anydesk
> 
> ![photo_2023-10-18_15-35-23](https://private-user-images.githubusercontent.com/62943818/276248605-9b3a40f8-ab6e-4f2a-8d62-6a035c4369f6.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE4ODE1MTMsIm5iZiI6MTcxMTg4MTIxMywicGF0aCI6Ii82Mjk0MzgxOC8yNzYyNDg2MDUtOWIzYTQwZjgtYWI2ZS00ZjJhLThkNjItNmEwMzVjNDM2OWY2LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzMxVDEwMzMzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJiNDVlNmY5MDMxMTdlZjRiNzQxYTg0NDM0OThiYjI1OTUwZmE1ZTIzNzMwOWM5Y2Y2MjEyZjI0ODJlNWJkNjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.NTmZROdlIiRu74l9oaF5hmHj9ZZe2g426XK4JhBJOAM) ![photo_2023-10-18_15-35-35](https://private-user-images.githubusercontent.com/62943818/276248612-68ba1fa0-b898-4663-83f5-c89acd2d3945.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE4ODE1MTMsIm5iZiI6MTcxMTg4MTIxMywicGF0aCI6Ii82Mjk0MzgxOC8yNzYyNDg2MTItNjhiYTFmYTAtYjg5OC00NjYzLTgzZjUtYzg5YWNkMmQzOTQ1LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzMxVDEwMzMzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFlOGFjNjU0NjE5MjcxZDkyOWU4OWY2NzM2NjIzNjdjMWQ2MTlmMmViM2IyYTQyYTIxMmQ5YjZkZDM1NDIxZDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.M3fbeeACFAh7c79cYJCjA0SAoYLWKJSsG_ZJ8h7xoI0) ![photo_2023-10-18_15-35-40](https://private-user-images.githubusercontent.com/62943818/276248617-29373858-658d-4a4f-bd20-6da32c64537f.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE4ODE1MTMsIm5iZiI6MTcxMTg4MTIxMywicGF0aCI6Ii82Mjk0MzgxOC8yNzYyNDg2MTctMjkzNzM4NTgtNjU4ZC00YTRmLWJkMjAtNmRhMzJjNjQ1MzdmLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzMxVDEwMzMzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgzYzdmZDZhZTM0NDc0OTA5ZWZmYTA1M2E3ZDI3NjAxMWVlOGI0YmMyNjRmZjM1Y2FhZWI1YWU5MjEwOTlhNWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Htf5U_DTP_VMFwPwAIQkHmTbCPgarDBejQ19koCpkyw)
> 
>  video1.mp4

Hi, how did you connect dremebot x 10 pro, without cloud? I constantly get an error, rebooting using budakt did not help. And if I use it through the cloud, I can’t configure the card, how did you configure it? Is it possible to send your settings to the Internet?
